protocol/chainlib/jsonRPC.go

```diff
@@ -484,6 +484,18 @@ func (cp *JrpcChainProxy) SendNodeMsg(ctx context.Context, ch chan interface{},
 			return nil, "", nil, utils.LavaFormatError("jsonRPC error", err, utils.Attribute{Key: "GUID", Value: ctx})
 		}
 		replyMsg = *replyMessage
+
+		reqId, idErr := rpcInterfaceMessages.IdFromRawMessage(nodeMessage.ID)
```

Comment:
let's create a common method that we use and add it to the error handler class. for both tendermint and jsonrpc

Reply:
fixed in: 92fb9142c18e1e95113212df7301f824f46e1e14